### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by darwinrisser**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 15 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of CSS Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 15 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 15 points

### CODE REVIEW

File: 02week/bio/blog.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.1/css/all.css" integrity="sha384-O8whS3fhG2OnA5Kas0Y9l3cfpmYjapjI0E4theH4iuMD+pLhbf6JI0jIMfYcK3yZ" crossorigin="anonymous">
```
Comment (situlillian) [inline 02week/bio/blog.html]: Love the use of font awesome!

File: 02week/bio/blog.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <link rel="stylesheet" type="text/css" href="css/blogstyle.css">
+    <meta charset="utf-8">
+    <title>Blog</title>
+  </head>
+  <body>
+    <header id="home">
+      <nav>
+        <a class="active" href="#home"><i class="fas fa-home"></i>  Home</a>
```
Comment (situlillian) [inline 02week/bio/blog.html]: Is this class being used for anything? I don't see any styling associated with it. If not used, no need to add a class name.

**Review by situlillian — APPROVED** [review]
Good use of semantic tags. Let's try to stick to the folder structure that is provided, mainly using the index.html (best practice). Go ahead and merge.

File: 02week/bio/blog.html (hunk 2 above)
Comment (darwinrisser) [reply 02week/bio/blog.html]: Thanks for pointing this out, I meant to use that class and then forgot to add the styling. It's updated now working correctly.

**Pull request merged**